src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String FILE_EXIST = "src/main/resourses/input.csv";
+    private static final String FILE_NOT_EXIST = "src/main/resourses/input_fake_file.csv";
+    private FileReader fileReaderTest;
+    private List<String> expectedList;
+    private List<String> actualList;
```

Comment:
do not make expected and actuals as fields
make them local variables in each test